src/eva/vision/utils/mask.py

```diff
@@ -0,0 +1,39 @@
+import cv2
+import numpy as np
+
+from eva.vision.data.wsi.backends.base import Wsi
+
+
+def get_mask(
+    wsi: Wsi,
+    level_idx: int,
+    kernel_size: tuple[int, int] = (7, 7),
+    gray_threshold: int = 220,
+    fill_holes: bool = False,
+) -> tuple[np.ndarray, float]:
```

Comment:
lets use `from typing import Tuple; Tuple[..]` to be consistent; this also helps to support older python versions, but at some point maybe we want to change everything to lowcase